projects/1.19/assets/heat-and-climate/dcs_climate/lang/zh_cn.json

```
@@ -0,0 +1,824 @@
+  "entity.dcs_climate.stick_beef":"牛肉串",
+  "entity.dcs_climate.stick_meat":"肉串",
+  "entity.dcs_climate.salad":"沙拉",
+  "entity.dcs_climate.soup":"汤碗",
+  "entity.dcs_climate.plate_steak":"烤牛排",
+  "entity.dcs_climate.plate_meat":"烤肉排",
+  "entity.dcs_climate.plate_legs":"烤腿排",
+  "entity.dcs_climate.plate_big_steak":"烤蒜香牛排",
+  "entity.dcs_climate.plate_chicken":"烤全鸡",
+  "entity.dcs_climate.plate_fish":"烤鱼排",
+
+    "##　tips":"",
+
+  "dcs.climate.tip.checker":"可以检测右击坐标的气候",
+  "dcs.tip.durability":"最大耐久度",
+  "dcs.tip.harvestlevel":"挖掘等级",
+  "dcs.tip.resistance":"气候抗性",
+
+  "dcs.name.shift":"按下左shift：展开工具提示",
+
+  "dcs.tip.null_item1":"没有物品被注册到这个配方",
+  "dcs.tip.null_item2":"请检查此配方的配置文件",
+
+  "dcs.tip.foodtaste.1":"风味等级：★",
+  "dcs.tip.foodtaste.2":"风味等级：★★",
+  "dcs.tip.foodtaste.3":"风味等级：★★★",
+  "dcs.tip.foodtaste.4":"风味等级：★★★★",
+  "dcs.tip.foodtaste.5":"风味等级：★★★★★",
+
+  "dcs.tip.raw_eaten_crop":"可生吃",
+  "dcs.tip.empty_pack":"右击：可以装取非高温液体",
+  "dcs.tip.remove_bad_potion":"移除负面药水效果",
+  "dcs.tip.scythe":"右击：收割范围内的热量与气候作物",
+
+  "## gui":"misc",
+
+  "dcs.smelting":"气候冶炼",
+  "dcs.recipe":"气候合成",
+  "dcs.mill":"石磨配方",
+  "dcs.fluidcraft":"流体合成",
+  "dcs.reactor":"反应配方",
+  "dcs.crop":"气候作物",
+  "dcs.effective":"气候影响方块",
+  "dcs.spinning":"纺织配方",
+  "dcs.biome":"群系气候数据",
+  "dcs.crusher":"粉碎机配方",
+
+  "death.attack.dcs_heat":"%1$s被烤的焦黄",
+  "death.attack.dcs_cold":"%1$s被冻成了冰雕",
+  "death.attack.dcs_water":"%1$s被淹成了泡菜",
+  "death.attack.dcs_dry":"%1$s被风干成了肉干",
+  "death.attack.dcs_wind":"%1$s乘风西去了",
+  "death.attack.dcs_suffocation":"%1$s死于氧气衰竭",
+  "death.attack.dcs_machine":"%1$s被卷进了机器",
+
+  "effect.dcs_climate.effect_cold_resistance":"严寒抗性",
+  "item.minecraft.potion.effect.cold_resistance":"严寒抗性药水",
+  "item.minecraft.splash_potion.effect.cold_resistance":"喷溅型严寒抗性药水",
+  "item.minecraft.lingering_potion.effect.cold_resistance":"滞留型严寒抗性药水",
+  "item.minecraft.tripped_arrow.effect.cold_resistance":"严寒抗性之箭",
+  "item.minecraft.potion.effect.long_cold_resistance":"严寒抗性药水",
+  "item.minecraft.splash_potion.effect.long_cold_resistance":"喷溅型严寒抗性药水",
+  "item.minecraft.lingering_potion.effect.long_cold_resistance":"滞留型严寒抗性药水",
+  "item.minecraft.tripped_arrow.effect.long_cold_resistance":"严寒抗性之箭",
+
+  "effect.dcs_climate.effect_wet":"潮湿",
+
+    "## gui":"jei",
+
+  "dcs.gui.jei.smelting_data":"气候冶炼数据",
+  "dcs.gui.jei.crop_data":"热量与气候作物数据",
+  "dcs.gui.jei.tree_data":"热量与气候树木数据",
+  "dcs.gui.jei.smelting_tip1":"对放置的方块生效",
+  "dcs.gui.jei.smelting_tip2":"对放置的实体生效",
+  "dcs.gui.jei.smelting_tip3":"对地上的掉落物生效",
+  "dcs.gui.jei.habitat":"栖息地",
+  "dcs.gui.jei.crop_soil":"适合的土壤",
+  "dcs.gui.jei.crop_farm":"农田",
+  "dcs.gui.jei.crop_dirt":"土或草方块",
+  "dcs.gui.jei.crop_sand":"沙子",
+  "dcs.gui.jei.crop_mud":"泥巴",
+  "dcs.gui.jei.crop_water":"水面",
+  "dcs.gui.jei.no_habitat":"没有野外分布",
+  "dcs.gui.jei.crop_tip1":"=作物变异=",
+  "dcs.gui.jei.crop_tip2":"“野生”等级的作物在施肥耕地上成长时有概率变异",
+  "dcs.gui.jei.tree_flower":"开花季节",

```

```suggestion
  "dcs.gui.jei.tree_flower":"花期",
```
https://www.termonline.cn/word/1356267339632431106/1#s1